doc/pages/Installation.md

```diff
@@ -201,41 +202,13 @@ For Fedora/CentOS/RHEL the required packages to build CRPropa:
   ```
 In case of CentOS/RHEL 7, the SWIG version is too old and has to be built from source.
 
-
 ### Mac OS X

```

Comment:
I remember that is was quite tricky to install crpropa on some Mac configuration, so I would suggest to keep the longer explanation for a little longer.

But maybe @reichherzerp, @sophieaerdker, or @LeanderSchlegel can comment on that.

Reply:
I adjusted the previous version now, @lukasmerten .
I think the reason for that complicated installation had something to do with AppleClang/LLVM and OpenMP, that is why all the flags were required. I think the newer versions of the compilers handle OpenMP much better.